gnocchi/rest/api.py

```diff
@@ -1302,20 +1311,21 @@ def _ResourceSearchSchema():
                     u"<=", u"≤", u"le",
                     u">=", u"≥", u"ge",
                     u"!=", u"≠", u"ne",
-                    u"like"
                 ): voluptuous.All(
                     voluptuous.Length(min=1, max=1),
                     {"id": _ResourceUUID,
-                     six.text_type: ResourceSearchSchemaAttributeValue},
+                     NotIDKey: ResourceSearchSchemaAttributeValue},
```

Comment:
```python
>>> voluptuous.NotIn(["id"])("foo")
'foo'
>>> voluptuous.NotIn(["id"])("id")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/voluptuous/validators.py", line 676, in __call__
    raise NotInInvalid(self.msg or 'value is not allowed')
voluptuous.error.NotInInvalid: value is not allowed
```
WDYT?

Reply:
Well, I may need glasses I have looks at voluptuous and haven't found this....